mage_integrations/mage_integrations/sources/dremio/README.md

```diff
@@ -0,0 +1,37 @@
+# Dremio
+
+![](https://www.dremio.com/wp-content/uploads/2023/02/CompanyDremio_centered.png)
+
+<br />
+
+## Config
+
+This source uses Dremio Arrow Flight Client to connect/execute SQL queries and shares most of its settings.
+To find more details, please refer to Dremio docs [here](https://docs.dremio.com/current/sonar/client-applications/clients/python)
+
+<b>NOTE:</b> Mage's implementation of Dremio Source doesn't allow for the `query` connection parameter
+
+| Key | Description | Required |
+| --- | --- | --- |
+| `username` | Username of the Dremio account to use for authenticating. | True |
+| `schema` | Dremio Source Table schema. | True |
+| `source_backend` | SQL Source backend used to query data from Dremio. Please refer to the section below for detailed info <br /> E.G postgresql, mssql, mysql. | True, if applicable |
+| `password` | Password of the Dremio account to use for authenticating. | True, if no token is provided. |
+| `hostname` | The hostname or IP address of the coordinator node. | False (Default is localhost) |
+| `port` | Dremio's Arrow Flight server port. Can be other than 32010, if changed on the coordinator node. | False (Default is 32010) |
+| `token` | Either a Personal Access Token or an OAuth2 Token. | True, if no password is provided. |
+| `tls` | Enables encryption on a connection. | False |
+| `disable_certificate_verification` | Disables TLS server verification. | False |
+| `path_to_certs` | Path to trusted certificates for encrypted connections. False |
```

Comment:
A pipe is missing before False in this line.

Reply:
Fixed in c78c0c2014babbe93ae7c3f20d2a522aeec2ff48
Thanks for the heads up!